simulations/uv_simulations.py

```diff
@@ -176,6 +175,11 @@ def get_baselines(self):
             y = baselines[1::2]
             x_base = np.append(x_base, x)
             y_base = np.append(y_base, y)
+            
+        drops = np.asarray([((i * 2 + np.array([1, 2])) - 1) + (i * self.len*2) for i in range(self.len)])
+        coords = np.delete(np.stack([x_base, y_base], axis=1), drops.ravel(), axis=0).T
+        x_base = coords[0]
+        y_base = coords[1]
```

Comment:
Warum ist das nötig?

Reply:
I delete all 0 length baselines. Before there were baselines from 1 antenna to the same antenna, which did not make sense.